vm_callinfo.h

```diff
@@ -363,6 +364,18 @@ vm_cc_attr_index_p(const struct rb_callcache *cc)
     return cc->aux_.attr_index > 0;
 }
 
+static inline uint32_t
+vm_ic_entry_index(const struct iseq_inline_iv_cache_entry *ic)
+{
+    return ic->entry->index;
+}
+
+static inline bool
+vm_ic_entry_p(const struct iseq_inline_iv_cache_entry *ic, VALUE obj)
```

Comment:
I think `foo_p(obj)` means `obj is foo`. And this function does it?

Reply:
Agreed this makes more sense, will refactor!